content/handbook/sales/public-sector.md

```
@@ -0,0 +1,148 @@
+## 政府顧客／契約
+
+[政府顧客／契約セクションをご覧ください](https://internal.gitlab.com/handbook/sales/public-sector/#government-customerscontracting)
+
+## GitLab for Campuses
+
+## 概要
+
+GitLab は、適格な教育機関に対し、キャンパスで GitLab を使用するための 2 つの異なるオプションを提供します:
+
+1. [**Academic Discount**](#academic-discount): GitLab ライセンスは、適格な教育機関に対し、定価から 20% の Academic Discount で利用可能です。これらのライセンスにユースケースの制限はありません。
+2. [**GitLab for Campuses**](#gitlab-for-campuses-enterprise-offering): 適格な教育機関は、エンタープライズキャンパス全体ライセンスを購入できます。キャンパス全体ライセンスは、機関の最後に公表された登録者数までの GitLab トップティア（self-managed または SaaS）の無制限のシートを提供します。このオファリングは、キャンパスの 1 つのライセンス内ですべてのユースケースが発生することを許可します。*注意:* 教育オプション（本書に記載）のいずれかに参加する能力は、GitLab の単独裁量に基づきます。クオリフィケーションに関する適用可能な手順に従ってください。**購入またはリクエストは、両当事者間で書面で締結されない限り、受諾された（または法的拘束力のある）ものとは見なされません。**
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)